libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -102,8 +102,8 @@ const swapProviders: Record<string, ProviderConfig> = {
       "3045022100e35f78bbdf543614fa7db7ccc45622186b727d1d5b0accbdaf220ac54d4cd2fb02200f7decbb5a6e822668af73bf0ddcde822121e29642436aa6de11feabf12cf2f0",
       "hex",
     ),
-    needsKYC: false,
-    needsBearerToken: false,
+    needsKYC: true,
+    needsBearerToken: true,
```

Comment:
Not sure that this property will be used for Moonpay integration.
If not, I recommend to set it as `false` for consistency (its goal should be if we need to keep the bearer token in Ledger Live, which should normally not be the case anymore with Moonpay).

Reply:
I don't see this property used anywhere except for checking if the quote card should say registration required or not.  I'll remove the bearer token part and keep the needsKYC as it will still show the desired text on the quote card